go.mod

```diff
@@ -84,4 +84,4 @@ require (
 	google.golang.org/protobuf v1.33.0 // indirect
 	gopkg.in/ini.v1 v1.66.4 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
-)
+)
```

Comment:
This file seems to have slight, inconsequential format change with respect to `master`. Can you bring `master`'s version so we don't have this change listed here?
```
$ git checkout --patch master go.mod
```

Reply:
fixed